### PR TITLE
Add experimental design offer

### PR DESCRIPTION
**Pull request by jenniferboedker**
This PR adds the property experimental design in the context of [#263](https://github.com/qbicsoftware/offer-manager-2-portlet/issues/263).

The offer builder has been adapted and also the test extended. Furthermore, also the changelog has been updated.

### CODE REVIEW

File: src/test/groovy/life/qbic/datamodel/dtos/business/OfferSpec.groovy [hunk 1]
```diff
@@ -37,7 +37,10 @@ class OfferSpec extends Specification {
         when:
         Offer testOffer =
                 new Offer.Builder(customer, projectManager, "Archer", "Cartoon Series", selectedAffiliation)
-                        .modificationDate(date).expirationDate(date).totalPrice(price).identifier(offerId).taxes(vat).overheads(overhead).netPrice(net).items([item]).itemsWithOverhead([item]).itemsWithoutOverhead([item]).itemsWithOverheadNet(itemsWithOverheadNet).itemsWithoutOverheadNet(itemsWithoutOverheadNet).overheadRatio(overheadRatio)
+                        .modificationDate(date).expirationDate(date).totalPrice(price).identifier(offerId).taxes(vat).overheads(overhead).netPrice(net).items([item])
+                        .itemsWithOverhead([item]).itemsWithoutOverhead([item]).itemsWithOverheadNet(itemsWithOverheadNet)
+                        .itemsWithoutOverheadNet(itemsWithoutOverheadNet).overheadRatio(overheadRatio)
+                        .experimentalDesign("design")
```
Comment (jenniferboedker) [inline src/test/groovy/life/qbic/datamodel/dtos/business/OfferSpec.groovy:43]: it looks strange but here I only added some spaces to improve the readability of the code

**Review by Steffengreiner — DISMISSED** [review]
LGTM 👍

**Review by Steffengreiner — APPROVED** [review]
Nice Idea with the optional 🎉

**Comment by jenniferboedker** [comment]
[Allow user to specify the experimental design separately](https://trello.com/c/ulTaYZlp/666-allow-user-to-specify-the-experimental-design-separately)

**Pull request merged**